app/uk/gov/hmrc/selfservicetimetopay/controllers/CalculatorController.scala

```diff
@@ -297,7 +297,9 @@ class CalculatorController @Inject() (val messagesApi: play.api.i18n.MessagesApi
       sessionCache.get.map {
         case Some(TTPSubmission(_, _, _, taxpayer, _, _, CalculatorInput(debits, paymentToday, _, _, _, _), _, _, _)) if debits.nonEmpty =>
           val form = CalculatorForm.createPaymentTodayForm(debits.map(_.amount).sum)
-          if (paymentToday.equals(BigDecimal(0))) Ok(payment_today_form(form, taxpayer.isDefined))
+          if (paymentToday.equals(BigDecimal(0))){
```

Comment:
Click testing this shows that the question page now doesn't work as intended because if you select Yes, the page always reloads the question without going to the form